integration-tests/cypress/workbench/Details.ts

```diff
@@ -19,4 +19,16 @@ export class Details {
   public getTextField(label: string): Cypress.Chainable<JQuery<HTMLInputElement | HTMLTextAreaElement>> {
     return this.getDetailsView().find(`[data-testid="input-${label}"]`);
   }
+
+  public getCheckBox(label: string): Cypress.Chainable<JQuery<HTMLInputElement>> {
+    return this.getDetailsView().findByTestId(`${label}`).find('input');
+  }
+
+  public openReferenceWidgetOptions(label: string): void {
+    this.getDetailsView().findByTestId(`${label}`).find(`button[title="Open"]`).click();
```

Comment:
Just `label`?

Reply:
No \` needed for the `'buton[title="Open"]'` string.

Reply:
done